### DISCUSSION TITLE
How to send comment using api?

### DISCUSSION THREAD

**Discussion by Mbrick2** [ScratchAddons/ScratchAddons#4087] [Help] [answered]
Though SAers would know

**Comment by TheColaber** [comment]
Depends where... But projects for example, would be a post request to `https://api.scratch.mit.edu/proxy/comments/project/projectid`

**Comment by Dhuls-Scratch** [comment, marked as answer]
Projects: `https://api.scratch.mit.edu/proxy/comments/project/$PROJECTID` Method: POST
Studios: `https://api.scratch.mit.edu/proxy/comments/studio/$PROJECTID` Method: POST
Profiles: `https://scratch.mit.edu/site-api/comments/user/$USERNAME/add/` Method: POST
Note that it should be formatted as JSON, like this:
`{"content":"Reply","parent_id":182344070,"commentee_id":44058862}`
`content` is the content of the message.
`parent_id` is the comment you are replying to; `""` for comments that aren't replies
`commentee_id` is the user ID of the person you are replying to; `""` for comments that aren't replies
